Toolset/palettes/message box/behaviors/revmessageboxbehavior.livecodescript

```diff
@@ -98,6 +96,38 @@ on openStack
    unlock screen
 end openStack
 
+on ideMsgChanged pTarget, pHandler, pLine, pMsg
```

Comment:
The content of `msg` isn't currently passed as a param is it? I think you need `msg`s here instead of `pMsg`